mass2/core/channel.py

```diff
@@ -45,6 +45,7 @@ class Channel:
     df: pl.DataFrame = field(repr=False)
     header: ChannelHeader = field(repr=True)
     npulses: int
+    subframediv: int = 64
```

Comment:
This should not have a default of a number that could be silently wrong. Require, or set it to None so it will break anywhere it is used. I prefer require, then we just update all the use sites.